src/sentry/integrations/github/client.py

```diff
@@ -594,6 +594,8 @@ def get_file(self, repo: Repository, path: str, ref: str) -> str:
     def get_blame_for_file(
         self, repo: Repository, path: str, ref: str, lineno: int
     ) -> Sequence[Mapping[str, Any]]:
+        if len(repo.name.split("/")) != 2:
+            raise ValueError("Invalid repo name")
```

Comment:
Another option here could be to use `.split("/", maxsplit=1)`. I'm not sure whether it's valid for repos to have a `/` in the name at all though.

Reply:
I added `maxsplit` too, but I'm not sure it's an alternative since the sentry issue is showing `split` is returning one value instead of two.